frontend/src/Components/Charts/Contraband/Contraband.js

```diff
@@ -47,28 +47,35 @@ function Contraband(props) {
 
   const renderMetaTags = useMetaTags();
   const [renderTableModal] = useTableModal();
-  const [contrabandData, setContrabandData] = useState({
+
+  const initContrabandData = {
     labels: [],
     datasets: [],
     isModalOpen: false,
     tableData: [],
     csvData: [],
     loading: true,
```

Comment:
This `loading` flag will show a Loading data component while data is being fetched.